modules/richards/test/tests/buckley_leverett/bl22_lumped_fu.i

```diff
@@ -136,6 +136,7 @@
 []
 
 [Bounds]
+  active = ''
```

Comment:
Does not apply bounds at all?

Reply:
It did not use the bounds, see its [Preconditioning] block. 

Our VI solver check will error out if it detects bounds in the input file but does not use VI solver.

Reply:
Delete these useless blocks?

Reply:
I prefer to just delete if we do not use at all

Reply:
It is not my code and I try to make minimum changes. 

It is up to you. I can delete them. Let me know.

Reply:
Yes. Just delete them. Thanks

Reply:
Done!